src/components/content-tab-snapshots.tsx

```diff
@@ -421,6 +447,8 @@ function AddDemoSiteWithProgress( {
 		tooltipContent = { text: allotmentConsumptionMessage };
 	} else if ( isAnySiteArchiving ) {
 		tooltipContent = { text: siteArchivingMessage };
+	} else if ( snapshotCreationBlocked ) {
+		tooltipContent = { text: userBlockedMessage };
```

Comment:
What about assigning the message directly?

```suggestion
		tooltipContent = { text: errorMessages.rest_site_creation_blocked };
```

Reply:
Yeah that's what I had initially https://github.com/Automattic/studio/pull/354/commits/5e6720ef8fe45640aecdf70b9867501611c4d6d2 but in one feedback comments, it was mentioned that we could pull it from the `errorMessages`. Let's go back to using the string directly to be consistent with other tooltip messages.

Reply:
Updated in https://github.com/Automattic/studio/pull/354/commits/e0ac36b0a0afd1c4be4ce79a780e760b1ff0f6b1

Reply:
@katinthehatsite , Sorry I meant to use the `errorMessages.rest_site_creation_blocked` directly without creating an intermediary variable.

Reply:
Alright, let's fix it up 👍